sourced/ml/cmd/merge_coocc.py

```diff
@@ -115,4 +115,4 @@ def merge_coocc_no_spark(df, filepaths, log, args):
             result.data[indx_overflow] = MAX_INT32
     Cooccurrences() \
         .construct(df.tokens(), result[:-1, :-1]) \
-        .save(args.output, (df,))
+        .save(output=args.output, series="merge-cooc-no-spark", deps=(df,))
```

Comment:
merge-cooc-no-spark -> id2vec (if I am not mistaken; depends on the problem we are solving with this command)